invenio_communities/communities/schema.py

```diff
@@ -75,9 +75,10 @@ def clean(self, data, **kwargs):
         Why: We want to allow the output of a Schema dump, to be a valid input
              to a Schema load without causing strange issues.
         """
-        for name, field in self.fields.items():
-            if field.dump_only:
-                data.pop(name, None)
+        if type(data) is dict:
```

Comment:
```suggestion
        if isinstance(data, dict):
```

Minor: See [here](https://switowski.com/blog/type-vs-isinstance#:~:text=Difference%20between%20isinstance%20and%20type&text=type%20only%20returns%20the%20type,specified%20as%20a%20second%20parameter.) why :)